specs/Standards/MultiToken/README.md

```diff
@@ -0,0 +1,539 @@
+- Proposal Name: Multi-Token-Standard
+- Start Date: 2021/07/24
+- Issue(s): #245.
+
+Version `1.0.0`
+
+# Summary
+[summary]: #summary
+
+A standard interface for  a multi token standard that supports fungible, semi-fungible, and tokens of any type, allowing for ownership, transfer, and batch transfer of tokens generally regardless of specific type.
+
+# Motivation
+[motivation]: #motivation
+Having a single contract represent both NFTs and FTs can greatly improve efficiency as demonstrated by Enjin Coin. The ability to make batch requests with multiple asset classes can reduce a many transactions transaciton to a single transaction to trade around both NFTs and FTs that are a part of same token contract.
+
+Having this will also increase Near's ability to work interoperably with other chains. This will reduce the complexity required to represent these emerging asset classes.
+
+
+Prior art:
+- EIP-1155 : https://github.com/ethereum/EIPs/issues/1155
+
+- This NEP derives some examples and format from: https://github.com/near/NEPs/pull/21
+
+
+- Things from the NFT Discussions: https://github.com/near/NEPs/discussions/171
+
+- Things from the NFT Discussions: https://gov.near.org/t/nft-standard-discussion/853
+
+Discussions out of band:
+ - https://gov.near.org/t/multi-token-standard-discussion/2917
+ - https://github.com/shipsgold/multi-token-standard-impl/tree/main/meetings 
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+We should be able to do the following:
+- Get balance of token per user
+- Get balance in batch of a user
```

Comment:
Great catch this could be more clear. 
```
Get balance of a single token_id per account
Get balance of multiple token_ids per account
```
Might be more clear. Token types aren't a thing that's exposed, in multi token standard it's only token_id that acts
as the interface for everything.

Reply:
If `token_id` is the 'key' for _everything_, how does someone identify what token_ids they own on a particular contract?